### DISCUSSION TITLE
Differences in gameplay. osu!lazer and osu!stable

### DISCUSSION THREAD

**Discussion by Norceosu** [ppy/osu#23605] [Q&A] [answered]
Hi,
So, after a long break from osu! I started to play lazer. At first it seemed like I got REALLY rusty from the break and had worse accuracy than ever. Only now, after downloading osu!stable and playing it again, I noticed the huge difference in accuracy and so I looked it up... So, what I understand from the changes with sliders, the stable version is broken? Devs mentioned that in stable sliders only required 50 / "Meh" worth of accuracy to get perfect and this didn't make sense for a rhythm game, so the change was done intentionally. Does this mean that these changes to sliders will remain for the future of osu!? I'm really torn between the two versions after finding this out. Yes, I'm late to the party and this is old news (I really wish I knew this sooner). The problem is that I spent 500hrs in stable learning to play the game wrong and now need guidance, whether I should focus on playing lazer instead of stable. The overall performance on stable is still better, but lazer's performance in gameplay is good enough, that I'm considering to switching to it permanently. I do believe that this change to sliders is the right choice, but I can't see myself improving my slider accuracy when stable just doesn't even care about them. Now, while this probably isn't relevant to the higher rank players (better overall accuracy skill), I want to know others opinion on it and maybe someone else from 6 digit rank who is also having slider accuracy issues in lazer can add to this.

TL;DR; Skill issue, probably not worth your time.

P.S. Another question. Will the lazer scores be ranked higher in pp (If, it isn't already), because of the need of higher skill in slider accuracy?

**Comment by integerrr** [comment, marked as answer]
https://github.com/ppy/osu/wiki/Gameplay-differences-from-osu!stable
https://docs.google.com/document/d/13cMCrQN4vvaQFA59BmFHk6D7PH9fwT2ANmhkm5cmzxE/edit

**Reply by peppy** [reply]
of note, you can also turn on classic mod if you want the old behaviour.

**Reply by AshuK2699** [reply]
so tell me which is better
osu or lazer

**Reply by RaoraRare** [reply]
> so tell me which is better
> osu or lazer

If you don't care about rank play osu!Lazer, if you do play osu!stable
